src/main/java/app/controllers/TimeZonesController.java

```diff
@@ -13,6 +13,6 @@ public static void addRoutes(Javalin app, ConnectionPool connectionPool)
 
     private static void index(Context ctx, ConnectionPool connectionPool)
     {
-        ctx.render("/timezones/index.html");
+        ctx.render("/timezones/startPage.html");
```

Comment:
please keep using the index as it will otherwise break the application for all other teams

Reply:
renamed to index.html